core/iwasm/fast-jit/jit_ir.def

```diff
@@ -92,9 +92,16 @@ INSN(I64TOF64, Reg, 2, 1)
 INSN(F32TOI32, Reg, 2, 1)
 INSN(F32TOI64, Reg, 2, 1)
 INSN(F32TOF64, Reg, 2, 1)
+INSN(F32TOU32, Reg, 2, 1)
 INSN(F64TOI32, Reg, 2, 1)
 INSN(F64TOI64, Reg, 2, 1)
 INSN(F64TOF32, Reg, 2, 1)
+INSN(F64TOU32, Reg, 2, 1)
+
+INSN(I32CASTF32, Reg, 2, 1)
+INSN(I64CASTF64, Reg, 2, 1)
+INSN(F32CASTI32, Reg, 2, 1)
+INSN(F64CASTI64, Reg, 2, 1)
```

Comment:
Can we add some comments about the difference between them and I32TOF32, I64TOF64?